tests/codeception/acceptance/Paragraphs/ListsCest.php

```diff
@@ -753,6 +753,8 @@ public function testListParagraphPeopleFiltersTypeFilter(AcceptanceTester $I) {
 
   /**
    * Test basic page types list view.
+   *
+   * @group D8CORE-7422
    */
   public function testListParagraphBasicPageTypesFilter(AcceptanceTester $I) {
```

Comment:
@buttonwillowsix Here's the test that i modified to test the sorting functionality. Basically I set the time it was created and made sure they were alphabetical.

Reply:
Ooh, I see! Thank you! I will try this next time I modify something like this on Sites.